editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -112,6 +118,9 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 		// refresh
 		try {
 			c.getProject().refreshLocal(IProject.DEPTH_INFINITE, subMonitor.newChild(1));
+			// update the manifest to reflect project name as Bundle-SymbolicName
+			updateBundleSymbolicName(c.getProject(), subMonitor.newChild(1));
+			c.getProject().refreshLocal(IProject.DEPTH_INFINITE, subMonitor.newChild(1));

```

Comment:
I'm not sure that we need to call the refresh local 2 times, or if needed, we need to wait for the job to be finished to be sure that it works everytime